### ISSUE TITLE
Error messages of pyright file-level type checking mode directives are not updated

### ISSUE THREAD

**Issue by NCBM** [microsoft/pyright#10988]
**Describe the bug**

The error messages:

- `"{directive}" is an unknown directive for pyright comment; expected "strict" or "basic" (pyrightCommentUnknownDirective)`
- `Pyright comment must be followed by a directive (basic or strict) or a diagnostic rule (pyrightCommentMissingDirective)`

are not updated. It still only includes `basic` and `strict` even when `standard` mode is already supported since 1.1.339 and the false positive of the message is also fixed in 1.1.383.

**Code or Screenshots**

Code sample in [pyright playground](https://pyright-play.net/?strict=true&locale=en-us&code=MQAgEgpgThIM4AsD2B3OIAOBPKBLA5ggC4gBmuANhALRUBuEFIRWGsAxghOwNa4B2%2BEAFskAE1hjcMdkVwM4AOgBQoAOT8k-CGpABDfmJBq9FCrr0wQAKwCucEqSRQQ7JMIyXccLQBpmXFj6VgISEEaaJHC2GBjOROEqqpg4BMQAXCCa2snYeIREmQBGenC47LmpBZkOBmKWYpX5GfBEeLJNaYX6Zsqd1UA)

```python
# Here shows pyright file-level type checking mode directives.
# 'none' and 'all' are just for comparison, they are indeed not supported.

# pyright: none
# pyright: basic
# pyright: standard
# pyright: strict
# pyright: all

# pyright:
```

**VS Code extension or command-line**

Reproducible in Pylance v2025.8.3 and Pyright 1.1.383 (the version that `standard` begins not to report error) and latest over pyright-play.net.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.407
